### PR TITLE
Remove RT.composeHOC

### PR DESCRIPTION
**Pull request by fiskus**
* Remove un-used code using RT.composeHOC
* Rewrite RT.loadable function with less flexibility but simple implementation

### CODE REVIEW

File: catalog/app/containers/App/App.js [hunk 1]
```diff
@@ -35,29 +35,36 @@ const LegacyPackages = ({ location: l }) => {
...
-const MPersonas = mkLazy(() => import('website/pages/Personas'))
-const MProduct = mkLazy(() => import('website/pages/Product'))
+const Admin = RT.mkLazy(() => import('containers/Admin'), Placeholder)
+const AuthActivationError = RT.mkLazy(
+  () => import('containers/Auth/ActivationError'),
+  Placeholder,
+)
+const AuthCode = requireAuth()(
+  RT.mkLazy(() => import('containers/Auth/Code')),
+  Placeholder,
```
Comment (nl0) [inline catalog/app/containers/App/App.js:45]: is there a typo here? like, `Placeholder`, but in other places you call `mkLazy` with two args: `() => import()` and `Placeholder`. is this intentional or a typo?

File: catalog/app/containers/App/App.js [hunk 2]
```diff
@@ -35,29 +35,36 @@ const LegacyPackages = ({ location: l }) => {
...
+const AuthSSOSignUp = RT.mkLazy(() => import('containers/Auth/SSOSignUp'), Placeholder)
+const Bucket = RT.mkLazy(() => import('containers/Bucket'), Placeholder)
+const Search = RT.mkLazy(() => import('containers/Search'), Placeholder)
+const UriResolver = RT.mkLazy(() => import('containers/UriResolver'), Placeholder)
+
+const Landing = RT.mkLazy(() => import('website/pages/Landing'), Placeholder)
+const OpenLanding = RT.mkLazy(() => import('website/pages/OpenLanding'), Placeholder)
+const OpenProfile = requireAuth()(
+  RT.mkLazy(() => import('website/pages/OpenProfile')),
+  Placeholder,
```
Comment (nl0) [inline catalog/app/containers/App/App.js:61]: same here

**Review by nl0 — APPROVED** [review]
ушла эпоха

File: catalog/app/containers/App/App.js (hunk 1 above)
Comment (fiskus) [reply catalog/app/containers/App/App.js:45]: Typo, thanks

**Pull request merged**